database/migrations/2023_12_16_000000_update_gamehashlibrary_table.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::table('GameHashLibrary', function (Blueprint $table) {
+            $table->string('compatibility')->nullable()->after('file_name_md5'); // Unverified, In Progress, Verified, Problematic
```

Comment:
Will those values be added as an Enum?
When stored in db it would be nice if they follow the same convention as types for achievements did; lowercase, snake- or kebap-case like unverified, in-progress, verified, problematic

Reply:
_HashCompatibility.php_ enum added in latest.